### PR TITLE
Make UAF POI consider uses of all allocation subregions

### LINKED ISSUE #52: UAF POI doesn't count uses of subregions
Hi there, I am crazy about MATE. When I tried to put a [sample C code with UAF vulnerabilities (Link attached)](https://github.com/yuawn/NTU-Computer-Security/blob/d6af31d7b71304ff6f44c8faa454815783923035/week3/src/uaf.c), MATE showed nothing and just an empty page in POI. Can you help me to figure it out? Thanks.

### PR DESCRIPTION
**Pull request by thinkmoore**
Fixes #52.

### CODE REVIEW

File: frontend/test/programs/ntu-uaf.c [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
```
Comment (langston-barrett) [inline frontend/test/programs/ntu-uaf.c:1]: Might be good to run `clang-format` on this (see #44)

File: tests/postgres/poi_analysis/use_after_free_test.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def test_poi_use_after_free_read(cpg_db_v2):
         for (free, usage, _, _) in compute_use_after_free(cpg.session, cpg)
     }
 
-    expected = {(20, 24)}
+    expected = {(20, 24), (20, 25)}
```
Comment (langston-barrett) [inline tests/postgres/poi_analysis/use_after_free_test.py:15]: I'm assuming that you checked that this new result is reasonable. If so, everthing LGTM!

**Review by langston-barrett — APPROVED** [review]
Thanks for the fix!

File: tests/postgres/poi_analysis/use_after_free_test.py (hunk 2 above)
Comment (thinkmoore) [reply tests/postgres/poi_analysis/use_after_free_test.py:15]: Yeah, we should have had these as expected results before...

File: frontend/test/programs/ntu-uaf.c (hunk 1 above)
Comment (thinkmoore) [reply frontend/test/programs/ntu-uaf.c:1]: Done.

**Comment by thinkmoore** [comment]
Don't squash commits. The second commit fixes an unrelated lint issue.

**Pull request merged**